### ISSUE TITLE
Build on windows using build-webkit fails if performed from tarball

### ISSUE THREAD

**Issue by tomga** [annulen/webkit#705]
Early in the build there is message:

Couldn't move qtwebkit-libs-win.headers to <qtwebkitsources>\WebKitLibraries\win.

and of course build fails later due to lack of required 3rdparty headers and libraries.

**Comment by annulen** [comment]
Is it enough to create the missing directory?

**Comment by tomga** [comment]
yes, I've added information about it on the wiki, as current tarballs will not be fixed

**Comment by xiaoli520** [comment]
![image](https://user-images.githubusercontent.com/3784901/76911010-946c8f80-68ea-11ea-819b-5d776bef7d10.png)
you need set env  WEBKIT_LIBRARIES
